dowhy/causal_estimator.py

```diff
@@ -149,8 +162,8 @@ def estimate_effect(self):
             signif_dict = self.test_significance(est)
             est.add_significance_test_results(signif_dict)
         if self._confidence_intervals:
```

Comment:
again, if it is a binary flag, better to call it something such that "name_of_variable=True" can be read off and it makes sense. 
E.g. "is_estimated=True" has a nice semantic meaning, and immediately conveys to the reader that a variable "is_estimated" probably has a binary value. 
One option for this variable is self._estimate_confidence_interval. But this also has a catch that it sounds like a method. Any other ideas?

Reply:
In any case, it should be singular "interval".